homeassistant/helpers/entity_component.py

```diff
@@ -263,6 +264,16 @@ def _async_update_group(self):
             self.logger.error(err)
             return None
 
+        # Make a copy because we are mutating it.
```

Comment:
The entity component should not be aware of packages…

Reply:
Potentially a yaml_config_loading function in `homeassistant.cofig`?

Reply:
What is your suggestion @balloob

Reply:
Make it a new function `async_hass_config_merged_yaml`?